src/indexes.rs

```diff
@@ -1795,9 +1795,20 @@ impl<'a, Http: HttpClient> AsRef<IndexUpdater<'a, Http>> for IndexUpdater<'a, Ht
 #[derive(Debug, Clone, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct IndexStats {
```

Comment:
Lets also add more doc-comments here

```suggestion
/// gives extended information and metrics about indexes and the Meilisearch database
pub struct IndexStats {
```